src/dune_rules/coq_stanza.ml

```diff
@@ -3,9 +3,7 @@ open Dune_lang.Decoder
 
 let coq_syntax =
   Dune_lang.Syntax.create ~name:"coq" ~desc:"the Coq language"
-    [ ((0, 1), `Since (1, 9))
-    ; ((0, 2), `Since (2, 5))
-    ; ((0, 3), `Since (2, 8))
```

Comment:
That doesn't seem like a good way of deleting old versions. The user will get a confusing message that `0.1` isn't recognized or some such. Instead, we should add a `Delete_in` constructor here that would give a proper message.